.github/workflows/graphql-schema-check.yml

```diff
@@ -42,7 +42,7 @@ jobs:
 
       - name: schema check
         env:
-          HIVE_TOKEN: ${{ secrets.hiveToken }}
+          HIVE_REGISTRY_TOKEN: ${{ secrets.hiveRegistryToken }}
         run: |
```

Comment:
Please move the `curl` to its own step (also in the other workflow for publish).